sql/executor.go

```diff
@@ -345,3 +350,13 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 		panic(fmt.Sprintf("unexpected type %T", t))
 	}
 }
+
+func getTimeZone(s Session) (*time.Location, error) {
```

Comment:
@vivekmenezes it's possible to have a method that takes its called by value. Why would we prefer a function?